miner/worker.go

```diff
@@ -1103,22 +1099,12 @@ loop:
 			log.Trace("Unknown circuit capacity checker error for L1MessageTx", "tx", tx.Hash().String(), "queueIndex", queueIndex)
 			log.Info("Skipping L1 message", "queueIndex", queueIndex, "tx", tx.Hash().String(), "block", w.current.header.Number, "reason", "unknown row consumption error")
 			w.current.nextL1MsgIndex = queueIndex + 1
-
-			// after `ErrUnknown`, ccc might not revert updates associated
-			// with this transaction so we cannot pack more transactions.
-			// TODO: fix this in ccc and change these lines back to `txs.Shift()`
-			circuitCapacityReached = true
-			break loop
+			txs.Shift()
```

Comment:
Is ccc in a consistent state after `circuitcapacitychecker.ErrUnknown`? Can we continue to pack more txs?

Reply:
`ErrUnknown` includes "trace txs.length mismatch" and "json.Marshal" error
(yeah we'd probably improve to more fine-grain error in the future)

so the state may or may not be consistent.

But if it become inconsistent it can easily fail for the next tx. So I think use `Shift()` to try to pack more txs here.

Reply:
> in the infra side, better don't assume it won't be consistent.

In that case we should keep
```go
circuitCapacityReached = true
break loop
```

Reply:
sorry typo

Reply:
Changed according to [this message](https://scrollco.slack.com/archives/C05M5U5HST0/p1692776298022329).